IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/Entities/IfcGeographicCRS/Documentation.md

```diff
@@ -0,0 +1 @@
+_IfcGeographicCRS_ is a coordinate reference system that is based on a geoid or ellipsoid. Geographic CRS typically requires two angles and a height (can be derived) to specify a location on a body such as ellipsoid, geoid etc
```

Comment:
Please delete geoid. This establishes (normal) height and uses an ellipsoid as base for localization. It is not part of / type of a *geographic* CRS.

Geographic CRS's height is "height above/below the ellipsoid's surface".